tests/hash-graph-http/tests/friendship.http

```diff
@@ -124,6 +124,7 @@ X-Authenticated-User-Actor-Id: {{account_id}}
     "$schema": "https://blockprotocol.org/types/modules/graph/0.3/schema/data-type",
     "kind": "dataType",
     "$id": "http://localhost:3000/@alice/types/data-type/meter/v/1",
+    "allOf": [{ "$ref": "https://blockprotocol.org/@blockprotocol/types/data-type/value/v/1" }],
```

Comment:
Similar comment to above (applies below also)... not a big deal as these are tests but it would be nicest if the inheritance structure was consistent across the example data types

Reply:
`length` is not available in this scenario, I just added it 😄 [`2e7e6e2` (#5240)](https://github.com/hashintel/hash/pull/5240/commits/2e7e6e204ea746842c641048e51122c88496e419)